### ISSUE TITLE
Return CI as attribute

### ISSUE THREAD

**Issue by DominiqueMakowski** [easystats/effectsize#174]
Related to https://github.com/easystats/parameters/pull/314

``` r
data <- iris
data$Cat1 <- rep(c("A", "B"), length.out = nrow(data))
x <- aov(Sepal.Length ~ Species * Cat1, data = data)
attributes(effectsize::effectsize(x))
#> $names
#> [1] "Parameter"    "Eta2_partial" "CI"           "CI_low"       "CI_high"     
#> 
#> $class
#> [1] "effectsize_table"     "see_effectsize_table" "data.frame"          
#> 
#> $row.names
#> [1] 1 2 3
```

<sup>Created on 2020-10-14 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

I wonder whether we could return the CI as a attribute under a name like `eta2_partial_ci` or something like that.

Essentially when it merges the two tables in report (https://github.com/easystats/report/blob/dev/R/report.aov.R#L93), I rename all the CI_low from effectsize as estimate + CI_low (e.g., `Eta2_partial_CI_low`) so that they can coexist with the main CIs. But I remove the CI columns, which is better off saved as an attribute. But we have to name differently than just `ci` as it can differ from the main one. 

TLDR; how to nicely print tables using `parameters::parameters_table()` that contain CIs related to the estimate + those of the effectsize

Not sure what's the best approach though

**Comment by mattansb** [comment]
> Essentially when it merges the two tables in report (https://github.com/easystats/report/blob/dev/R/report.aov.R#L93), I rename all the CI_low from effectsize as estimate + CI_low (e.g., Eta2_partial_CI_low) so that they can coexist with the main CIs. But I remove the CI columns, which is better off saved as an attribute. But we have to name differently than just ci as it can differ from the main one.

Why not do this on your end? This can easily be done with `is_effectsize_name()`:

``` r
data <- iris
data$Cat1 <- rep(c("A", "B"), length.out = nrow(data))
x <- aov(Sepal.Length ~ Species * Cat1, data = data)

out <- effectsize::effectsize(x)

es_name <- effectsize::is_effectsize_name(names(out))
es_name <- names(es_name)[es_name]

attr(out, paste0(es_name, "_ci")) <- out$CI[1]
out$CI <- NULL

attributes(out)
#> $names
#> [1] "Parameter"    "Eta2_partial" "CI_low"       "CI_high"     
#> 
#> $row.names
#> [1] 1 2 3
#> 
#> $Eta2_partial_ci
#> [1] 0.9
#> 
#> $class
#> [1] "effectsize_table"     "see_effectsize_table" "data.frame"
```

<sup>Created on 2020-10-14 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by DominiqueMakowski** [comment]
okay
